synapse/server_notices/resource_limits_server_notices.py

```diff
@@ -91,24 +92,28 @@ def maybe_send_server_notice_to_user(self, user_id):
         currently_blocked, ref_events = yield self._is_room_currently_blocked(room_id)
 
         try:
-            if not self._config.mau_limit_alerting:
-                # Alerting disabled, reset room if necessary and return
+            is_auth_blocking = False
+            event_limit_type = ""
```

Comment:
`None` would be a more conventional sentinel here

```suggestion
            event_limit_type = None
```

Reply:
shouldn't it be called `limit_type` rather than `event_limit_type`?